_rules/SC3-1-1-html-has-lang.md

```diff
@@ -99,12 +99,36 @@ The `lang` attribute specified has a non-empty value. The rule does not verify t
 
 #### Passed example 7
 
-The `xml:lang` attribute specified has a non-empty value. The rule does not verify the validity of the value specified and checks only for presence of a value.
+The `xml:lang` attribute specifies a value that is not the empty string (""). The rule does not verify the validity of the value specified and checks only for presence of a value.
```

Comment:
`attribute specifies a value that is not the empty string ("")` -> `attribute is not empty ("")`. The same should be done to all other uses of "is not the empty string"